plugins/builds/index.js

```diff
@@ -24,11 +24,8 @@ const workflowParser = require('screwdriver-workflow-parser');
  * @param  {Boolean}  [config.start]        Whether to start the build or not
  * @return {Promise}
  */
-function createBuild(config) {
-    const {
-        jobFactory, buildFactory, pipelineId, jobName, username, scmContext, build, start
-    } = config;
-
+function createBuild({ jobFactory, buildFactory, pipelineId, jobName, username,
```

Comment:
Did you want to add `start=true` to this so it has a predefined value if not passed?